firetweet/src/main/java/org/getlantern/firetweet/util/CustomTabUtils.java

```diff
@@ -183,6 +186,12 @@ private static String getTagByType(@NonNull String type) {
             case TAB_TYPE_DIRECT_MESSAGES: {
```

Comment:
A bit nitpicking: as we just return `type` or null, table lookup might be more error proof than switch statement here.